### ISSUE TITLE
Has anti-aircraft gun intercepts been implemented?

### ISSUE THREAD

**Issue by samyelias** [yairm210/Unciv#3081]
I know the anti-aircraft gun units exist, I built a few to deal with an enemy Triplane hitting my units. But I can't tell if they were doing anything. Are they fully implemented? If so maybe there should be a line in the game log about your intercept-type units hitting back, since we can't see animations of it happening?

**Comment by yairm210** [comment]
There are notifications in place.
I'm pretty sure the interception is working, send me the game file if you're unsure and I'll check it out

**Comment by samyelias** [comment]
I saw notifications for the Triplane hitting my units (and city), but I don't think there were notifications for the interceptions? Also even with notifications I have no idea how much damage I'm doing to the Triplane. That said I don't remember if the original Civ V gives you this info either...

I'll try to dig up a save file, I might have one for this.

**Comment by samyelias** [comment]
OK, I've attached a save file. Look for my city Tokyo, roughly in the middle of my lands. There's an Anti-Aircraft Gun unit fortified in the city. There's a worker in the hex just north of Tokyo. If you press Next Turn, an enemy Triplane (from the city of Jenne I believe) destroys that worker. On the next turn the Triplane destroys a work in the hex just south of Tokyo.

I can't tell if the Anti-Aircraft Gun is doing anything. Later on in the game as I advanced on Jenne to take it, I had two Anti-Aircraft Guns to try to protect my units, but it didn't seem to help (or maybe it just didn't help fast enough).

[Rome368turns.zip](https://github.com/yairm210/Unciv/files/5164742/Rome368turns.zip)
